pwa-shell/src/auth/default/withDefaultAuth.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import DefaultAuthContext from 'auth/default/DefaultAuthContext';
+import useKeycloak from 'auth/useKeycloak';
+
+const withDefaultAuth = WrappedComponent => {
+  const Context = DefaultAuthContext;
+  return class ComponentWithDefaultAuth extends React.Component {
+    renderWrappedComponent = ({ authInitialized, auth }) => (
+      <WrappedComponent
+        {...this.props}
+        authInitialized={authInitialized}
+        useKeycloak={useKeycloak}
```

Comment:
`useKeycloak` should not be used anymore at this point: the `auth` object should be the keycloak object or the default auth object